### PR TITLE
RS 7.8.2 release docs

### PR DESCRIPTION
**Pull request by rrelledge**
Epics: [DOC-3230](https://redislabs.atlassian.net/browse/DOC-3230), [DOC-3943](https://redislabs.atlassian.net/browse/DOC-3943), [DOC-4266](https://redislabs.atlassian.net/browse/DOC-4266), [DOC-4260](https://redislabs.atlassian.net/browse/DOC-4260), [DOC-3944](https://redislabs.atlassian.net/browse/DOC-3944)

Most of the docs in this PR have already been reviewed in previous PRs. The release notes are the main docs left to review:
1. [RS 7.8.2-34 detailed release notes](https://redis.io/docs/staging/release-rs-fuya-fuya/operate/rs/release-notes/rs-7-8-releases/rs-7-8-2-34/)
2. [RS 7.8 release notes index page](https://redis.io/docs/staging/release-rs-fuya-fuya/operate/rs/release-notes/rs-7-8-releases/)

[DOC-3230]: https://redislabs.atlassian.net/browse/DOC-3230?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOC-3943]: https://redislabs.atlassian.net/browse/DOC-3943?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOC-4266]: https://redislabs.atlassian.net/browse/DOC-4266?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOC-4260]: https://redislabs.atlassian.net/browse/DOC-4260?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOC-3944]: https://redislabs.atlassian.net/browse/DOC-3944?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by andy-stark-redis — COMMENTED** [review]
Aside from the missing info you mentioned, the 7.8.2 index page and release notes look fine.

File: content/operate/rs/references/rest-api/requests/bdbs/availability.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+|------|-------------|
+| no_quorum | Master healthcheck failed (no quorum in the cluster) |
+| db_not_found | Database does not exist in the cluster |
+| bdb_endpoint_unavailable | Local database endpoint is not available | 
+
+### Status codes {#get-endpoint-status-codes}
+
+| Code | Description |
+|------|-------------|
+| [200 OK](https://www.rfc-editor.org/rfc/rfc9110.html#name-200-ok) | Database endpoint is available |
```
Comment (mich-elle-luna) [inline content/operate/rs/references/rest-api/requests/bdbs/availability.md:76]:
```suggestion
| [200 OK](https://www.rfc-editor.org/rfc/rfc9110.html#name-200-ok) | Database endpoint is available. |
```

File: content/operate/rs/references/rest-api/requests/bdbs/availability.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+|------|-------------|
+| no_quorum | Master healthcheck failed (no quorum in the cluster) |
+| db_not_found | Database does not exist in the cluster |
+| bdb_unavailable | Database is not available | 
+
+### Status codes {#get-db-status-codes}
+
+| Code | Description |
+|------|-------------|
+| [200 OK](https://www.rfc-editor.org/rfc/rfc9110.html#name-200-ok) | Database is available |
```
Comment (mich-elle-luna) [inline content/operate/rs/references/rest-api/requests/bdbs/availability.md:141]:
```suggestion
| [200 OK](https://www.rfc-editor.org/rfc/rfc9110.html#name-200-ok) | Database is available. |
```

**Review by mich-elle-luna — APPROVED** [review]
thank you!

**Review by bascar — APPROVED** [review]
LGTM!

**Pull request merged**